demo/vl-map-select-action.html

```diff
@@ -10,9 +10,9 @@
     <script>
```

Comment:
Ik zie hier in de demo-pagina ook bij een bepaalde select actie dat er een console.log gewoon gebeurt als er een feature geselecteerd wordt. Dat lijkt mij altijd een beetje nutteloos, iemand die in de demo pagina wat rondklikt ziet dan nog niets veranderen, tenzij hij de console open zet. Ik zou da altijd op het scherm ergens tonen dan.

Reply:
en er zit wat duplicatie in die code.

Reply:
Ok, heb het veranderd naar een tekst zichtbaar op de demo pagina, waarop ook getest wordt